### PR TITLE
GCG-35 add the q10.json

### PR DESCRIPTION
**Pull request by prncoprs**

### CODE REVIEW

File: codegen/src/test/resources/q10/q10.json [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+      {
+        "name": "Q10Orders",
+        "relation": "orders",
+        "this_key": [
+          "custkey"
+        ],
+        "next_key": [
+          "orderkey"
+        ],
+        "child_nodes": 2,
```
Comment (wqc0712) [inline codegen/src/test/resources/q10/q10.json:64]: It only has one child node.

File: codegen/src/test/resources/q10/q10.json [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+        "next_key": [
+          "custkey", 
+          "c_name",
+          "c_acctbal",
+          "c_phone",
+          "c_address",
+          "c_comment",
+          "n_name"
+        ],
+        "child_nodes": 3,
```
Comment (wqc0712) [inline codegen/src/test/resources/q10/q10.json:32]: It only has one child node.

File: codegen/src/test/resources/q10/q10.json [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+        "is_Root": false,
+        "is_Last": true,
+        "rename_attribute": null,
+        "select_conditions": {}
+      }
+    ],
+    "AggregateProcessFunction": [
+      {
+        "name": "Q10Aggregate",
+        "this_key": [
```
Comment (wqc0712) [inline codegen/src/test/resources/q10/q10.json:134]: this_key here should be all attributes in the group_by clause.

File: codegen/src/test/resources/q10/q10.json (hunk 1 above)
Comment (prncoprs) [reply codegen/src/test/resources/q10/q10.json]: revised in `https://github.com/DanishAlsayed/gui-codegen/pull/26/commits/92f88668c395efd0e64745a9f0a0b3bccbb345fa`

File: codegen/src/test/resources/q10/q10.json (hunk 3 above)
Comment (prncoprs) [reply codegen/src/test/resources/q10/q10.json:134]: revised in [https://github.com/DanishAlsayed/gui-codegen/pull/26/commits/9a8adc0d4669d8d2114450b9a177fd4ed20ff067](url)

**Comment by prncoprs** [comment]
@wqc0712 the q10.json has been revised, could u plz merge this branch. thx.

**Review by wqc0712 — APPROVED** [review]

**Pull request merged**